app/boards/makerbase/mks_canable_v20/mks_canable_v20.conf

```diff
@@ -0,0 +1,3 @@
+# The three lines below are needed to reduce ROM usage
```

Comment:
```suggestion
# The two lines below are needed to reduce flash usage
```

Reply:
Strage. I've applied the suggestion and github resolved this automatically. Thus it was not me 😄